Packs/ThalesCipherTrustManager/Integrations/ThalesCipherTrustManager/ThalesCipherTrustManager.yml

```diff
@@ -0,0 +1,2388 @@
+      description: Count for the number of logins
+      type: Number
+    - contextPath: CipherTrust.Users.last_login
+      description: Timestamp of last login
+      type: Date
+    - contextPath: CipherTrust.Users.created_at
+      description: Timestamp of when user was created
+      type: Date
+    - contextPath: CipherTrust.Users.updated_at
+      description: Timestamp of last update of the user
+      type: Date
+    - contextPath: CipherTrust.Users.allowed_auth_methods
+      description: List of login authentication methods allowed to the user.
+      type: Unknown
+    - contextPath: CipherTrust.Users.expires_at
+      description: The expires_at is applicable only for local user accounts. The admin or a user who is part of the admin group can add expiration to an existing local user account or modify the expiration date. Once the expires_at date is reached, the user account gets disabled and the user is not able to perform any actions.
+      type: Date
+    - contextPath: CipherTrust.Users.password_policy
+      description: The password policy applies only to local user accounts and overrides the global password policy. By default, the global password policy is applied to the users.
+      type: String
+    - contextPath: CipherTrust.Users.allowed_client_types
+      description: List of client types allowed to the user.
+      type: Unknown
+    - contextPath: CipherTrust.Users.failed_logins_count
+      description: Count of failed login attempts
+      type: Number
+    - contextPath: CipherTrust.Users.failed_logins_initial_attempt_at
+      description: Timestamp of the initial failed login attempt.
+      type: Date
+    - contextPath: CipherTrust.Users.account_lockout_at
+      description: Timestamp of when the account was locked.
+      type: Date
+    - contextPath: CipherTrust.Users.last_failed_login_at
+      description: Timestamp of the last failed login attempt.
+      type: Date
+    - contextPath: CipherTrust.Users.password_changed_at
+      description: Timestamp of when the password was last changed.
+      type: Date
+    - contextPath: CipherTrust.Users.password_change_required
+      description: Indicates if a password change is required at next login.
+      type: Boolean
+    - contextPath: CipherTrust.Users.login_flags
+      description: Flags related to login, such as prevent_ui_login.
+      type: Unknown
+  - deprecated: false
+    description: 'Returns a list of user  Command arguments can be used to filter the results. The results can be filtered, using the command arguments. '
+    name: ciphertrust-users-list
+    arguments:
+    - name: name
+      isArray: false
+      description: Filter by the user's name
+      required: false
+      secret: false
+      default: false
+    - name: user_id
+      isArray: false
+      description: If provided, gets the user with the specified user_id.  If the user_id 'self' is provided, it will return the current user's information.
```

Comment:
```suggestion
      description: If provided, gets the user with the specified user ID.  If the user ID 'self' is provided, it will return the current user's information.
```